tests/test_writers.py

```diff
@@ -191,3 +192,69 @@ def test_write_sssom_ontoportal_json(self):
             self.mapping_count,
             f"{path} has the wrong number of mappings.",
         )
+
+    # TODO: Determine assertions that should pass
+    # TODO: Fix failing assertions that should pass
+    # TODO: before merging, remove pass/fail comments at end of assertion lines
+    def test_missing_entries(self):
+        """Test to make sure that missing entires do not appear"""
+        df = pd.DataFrame(
+            [
+                {
+                    "subject_id": "Orphanet:58",
+                    "subject_label": "Alexander disease",
+                    "predicate_id": "skos:exactMatch",
+                    "object_id": "icd11.foundation:2023359698",
+                    "object_label": "Alexander disease",
+                    "mapping_justification": "semapv:UnspecifiedMatching",
+                }
+            ]
+        )
+        metadata = {
+            "creator_id": "orcid:0000-0002-2906-7319",
+            "curie_map": {
+                "MONDO": "http://purl.obolibrary.org/obo/MONDO_",
+                "Orphanet": "http://www.orpha.net/ORDO/Orphanet_",
+                "icd11.foundation": "http://id.who.int/icd/entity/",
+                "oboInOwl": "http://www.geneontology.org/formats/oboInOwl#",
+                "orcid": "https://orcid.org/",
+                "owl": "http://www.w3.org/2002/07/owl#",
+                "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+                "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+                "semapv": "https://w3id.org/semapv/",
+                "skos": "http://www.w3.org/2004/02/skos/core#",
+                "sssom": "https://w3id.org/sssom/",
+            },
+            "license": "http://w3id.org/sssom/license/unspecified",
+            "mapping_provider": "https://www.orpha.net/",
+        }
+
+        # When passing Converter
```

Comment:
## FYI: 2 different setups
I have a set of assertions for 2 cases of `MappingSetDataFrame` instantiation:
- with inclusion of a `Converter`
- with no inclusion of a `Converter`